docs/content/latest/api/ycql/dml_insert.md

```diff
@@ -63,6 +63,8 @@ Where
 - The columns list must include all primary key columns.
 - The `USING TIMESTAMP` clause indicates you would like to perform the INSERT as if it was done at the
   timestamp provided by the user. The timestamp is the number of microseconds since epoch.
```

Comment:
Is this correct? Here it says milliseconds: https://cassandra.apache.org/doc/latest/cql/types.html#working-with-timestamps

Reply:
@stevebang  That is for `timestamp` column. While this is for versioning the row. See docs:

1. https://docs.datastax.com/en/cql-oss/3.3/cql/cql_reference/cqlInsert.html#cqlInsert__timestamp-value
2. https://docs.scylladb.com/getting-started/dml/#update-parameters